doc/ExpertOptions.md

```diff
@@ -1475,28 +1475,29 @@ You would write the following manifest:
 }
 ```
 
-When you compile the workflow, provide the manifest using the `-inputs` option, and it will be translated to:
-
-`mymanifest.dx.json`
+Compile the workflow `test` from above with the `-inputs mymanifest.json` option. A new file `mymanifest.dx.json` will be 
+created with the following content. **NOTE** `mymanifest.dx.json` is created by the compiler - the user does not need to 
+create/change it manually.
 ```json
 {
   "input_manifest___": {
-    "s": "hello",
-    "f": {
-      "$dnanexus_link": "file-xxx"
-    }
-  },
-  "input_manifest___files": [
-    {
-      "$dnanexus_link": "file-xxx"
+    "encoded": false,
+    "types": {
+      "f": "File",
+      "s": "String"
+    },
+    "values": {
+      "s": "hello",
+      "f": "dx://file-xxx"
     }
-  ]
+  }
 }
 ```
 
-Finally, run your workflow using the translated input file:
-
-`dx run workflow-yyy -f mymanifest.dx.json`
+The created `mymanifest.dx.json` should be used as an input file when running the workflow:
+```commandline
+dx run workflow-yyy -f mymanifest.dx.json
+```
 
 ## Manifest file
 
```

Comment:
Here it says " Given the above workflow, the manifest output would be". Does that mean that a workflow compiled with `-useManifests` will always have an additional output added by the dxCompiler? If so, perhaps it's good to mention it in the intro section.

Also, in the `Manifest JSON` section we say that "each applet has an `input_mainfest___` input field of type `hash`". Perhaps, for symmetry, we could say something corresponding in `Manifest file`, with `input_manifest_files___` (since we mention it below but don't introduce anywhere at the beginning of this section).

Reply:
Fixed in [34af2d0](https://github.com/dnanexus/dxCompiler/pull/354/commits/34af2d06912bb083b83ecf159a95c62e9e91a92f)[34af2d0](https://github.com/dnanexus/dxCompiler/pull/354/commits/34af2d06912bb083b83ecf159a95c62e9e91a92f)

> will always have an additional output added by the dxCompiler?

Point 1 in the manifest intro.

> Perhaps, for symmetry, we could say something corresponding in Manifest file...

Added point 2 in the intro. Tried to separate these 2 cases more explicitly.